lib/routes.js

```diff
@@ -74,23 +75,55 @@ function fastifySwagger (fastify, opts, done) {
     }
   })
 
-  fastify.route({
-    url: '/uiConfig',
-    method: 'GET',
-    schema: { hide: true },
-    ...hooks,
-    handler: (req, reply) => {
-      reply.send(opts.uiConfig)
-    }
-  })
+  const uiConfig = serialize(opts.uiConfig)
+  const initOAuth = serialize(opts.initOAuth)
+
+  const swaggerInitializer = `window.onload = function () {
+  function resolveUrl(url) {
+    const anchor = document.createElement('a')
+    anchor.href = url
+    return anchor.href
+  }
+
+  function resolveConfig(cb) {
+    const config = ${uiConfig}
+    const resConfig = Object.assign({}, {
+      dom_id: '#swagger-ui',
+      deepLinking: true,
+      presets: [
+        SwaggerUIBundle.presets.apis,
+        SwaggerUIStandalonePreset
+      ],
+      plugins: [
+        SwaggerUIBundle.plugins.DownloadUrl
+      ],
+      layout: "StandaloneLayout"
+    }, config, {
+      url: resolveUrl('./json').replace('static/json', 'json'),
+      oauth2RedirectUrl: resolveUrl('./oauth2-redirect.html')
+    });
+    return cb(resConfig);
+  }
+
+  // Begin Swagger UI call region
+  const buildUi = function (config) {
+    const ui = SwaggerUIBundle(config)
+    window.ui = ui
+    ui.initOAuth(${initOAuth});
+  }
+  // End Swagger UI call region
+  resolveConfig(buildUi);
+}`
 
   fastify.route({
-    url: '/initOAuth',
+    url: '/static/swagger-initializer.js',
     method: 'GET',
     schema: { hide: true },
     ...hooks,
     handler: (req, reply) => {
-      reply.send(opts.initOAuth)
+      reply
+        .header('content-type', 'application/javascript; charset=UTF-8')
```

Comment:
```suggestion
        .header('content-type', 'application/javascript; charset=utf-8')
```

I know charset is case insensitive, but maybe make this lowercase to keep in line with what Fastify uses?